src/sentry/preprod/authentication.py

```diff
@@ -0,0 +1,16 @@
+from sentry.api.authentication import AuthenticationSiloLimit, ServiceRpcSignatureAuthentication
+from sentry.silo.base import SiloMode
+
+LAUNCHPAD_RPC_SHARED_SECRET_SETTING = "LAUNCHPAD_RPC_SHARED_SECRET"
+
+
+@AuthenticationSiloLimit(SiloMode.CONTROL, SiloMode.REGION)
```

Comment:
```suggestion
@AuthenticationSiloLimit(SiloMode.REGION)
```

I don't think you'll be sending requests to control silo.